article/app/controllers/LiveBlogController.scala

```diff
@@ -39,6 +39,14 @@ class LiveBlogController(
   // we support liveblogs and also articles, so that minutes work
   private def isSupported(c: ApiContent) = c.isLiveBlog || c.isArticle
 
+  private[this] val logger = DotcomponentsLogger()
+
+  private[this] def logRequest(msg: String, results: Map[String, String], page: PageWithStoryPackage)(implicit
+      request: RequestHeader,
+  ): Unit = {
+    logger.withRequestHeaders(request).results(msg, results, page)
+  }
+
```

Comment:
This might be more faff than it's work but rather than duplicate the logic in articlepicker https://github.com/guardian/frontend/blob/4dabb0b00619d84779fe16c8efcfbcfd804554a4/article/app/services/dotcomponents/ArticlePicker.scala#L51-L57 it might be nicer to put these two functions into the DotcomponentsLogger object https://github.com/guardian/frontend/blob/fa152a31bc3a4336c960c58b4b9e8ab52dd10730/article/app/services/dotcomponents/DotcomponentsLogger.scala#L101 and import them from there